src/components/NewTodos/NewTodos.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './NewTodos.css';
+
+import { NewTodosProps } from '../PropTypes/PropTypes';
+
+class NewTodos extends React.Component {
+  state = {
+    inputTitle: '',
+    selectedUser: '',
+    errorTitle: '',
+    errorUser: '',
+  };
+
+  handleChangeTitle = (event) => {
```

Comment:
если ивент не используем, то упростим
```suggestion
  handleChangeTitle = ({ target }) => {
```